tests/integration/test_github.py

```diff
@@ -210,6 +210,12 @@ def test_issue_info(self):
         assert issue_info.title.startswith("add basic readme")
         assert issue_info.status == IssueStatus.closed
 
+    def test_issue_labels(self):
+        self.colin_project.add_issue_labels(issue_id=2, labels=["test_lb1", "test_lb2"])
```

Comment:
Is it possible to also test the label-number change to be sure that we change something? (We can have these labels already there.)

Reply:
>Is it possible to also test the label-number change to be sure that we change something? 

I guess you mean labels count on Issue and PR. I added it now in the second commit.